samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -134,6 +146,11 @@ var AppController = function(loadingParams) {
       this.roomSelection_.onRoomSelected = function(roomName) {
         this.hide_(roomSelectionDiv);
         this.finishCallSetup_(roomName);
+
+        this.roomSelection_ = null;
+        if (this.localStream_) {
```

Comment:
Chuck is working on showing the self preview in the room selection UI. So it will be displayed right away with that change.